### ISSUE TITLE
Threnody not working as intended on enspells

### ISSUE THREAD

**Issue by greyenvy** [EdenServer/community#3441]
### Checklist

- [X] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [X] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [X] I have provided my client version in the details. (type /ver into your game window)

### Details
 /ver 30181205_0

At the very least Lightning Threnody does not work as it should.  In the following screenshots my RDM is meleeing a Devil Manta with Enthunder on (weapon is Joyeuse).  I will show pictures before I apply Lightning Threnody and after.  Please note the change in additional effect damage. It appears to be averaging lower only after Threnody is applied, which I do not believe is accurate.  Lightning Threnody lowers the mob's resistance to Lightning as per the tooltip.  It should not be increasing it!

Screenshots:
![Ashton_2020 11 23_134138](https://user-images.githubusercontent.com/48332558/100002487-9c8ea000-2d92-11eb-9bbb-14eab40974e6.png)
![Ashton_2020 11 23_134144](https://user-images.githubusercontent.com/48332558/100002490-a1535400-2d92-11eb-9d0b-65965b5a69a7.png)

EDIT: Added Ice Threnody screenshots as well.  Same exact problem!
![Ashton_2020 11 23_135150](https://user-images.githubusercontent.com/48332558/100003260-c6949200-2d93-11eb-864b-c6bed7051ecf.png)
![Ashton_2020 11 23_135200](https://user-images.githubusercontent.com/48332558/100003268-c98f8280-2d93-11eb-8b00-67abbbd27a03.png)

Something's up with at least Lightning and Ice Threnody.  May also be an interaction with the enspell?  Either way it is doing exactly the opposite of what it intends right now.  Please look into it!  Thank you!

**Comment by E5una** [comment]
I did a little testing on this with my 45 BLM with uncapped (but close) elemental skill vs IT cactaur in West Altepa + my BRD using threnody. I was able to land 25 full damage Thunder's on a Cactaur with Lightning Threnody on, so I don't think this strange behavior is affecting ALL aspects of resists. I switched to 75SMN and tested on the higher level Antican Hplomachus next to the cactaur with Rolling Thunder from Ramuh and noticed the additional damage from Rolling Thunder would vary WILDLY once I put the lightning threnody on. Before it I only saw ~15 per hit, but after it there were a lot of procs for 0 and 3 mixed in witht he procs for 15. The issue might be localized to enspells.

![image](https://user-images.githubusercontent.com/13668758/104326150-62e73500-54b7-11eb-955e-60a195d38aeb.png)

Data gathered on test server, not live.
